keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -345,8 +342,11 @@ spec:
   backoffLimit: 3
   backoffPolicy:
     rules:
-    - containerExitCodeNotIn: 40-50
-      action: Terminate
+    - action: Terminate
+      onContainerExitCode:
+        notIn:
+          min: 40
+          max: 50
```

Comment:
I think optional max could be misleading as it can be interpreted as interval which is only bounded from left. 
But I get the point about API to express a single exit code easy - let me think and come up with something

Reply:
If someone wants to target a single exit code (say 42), it can be spelled in the new API:

```
    - action: Terminate
      requirements:
      - source: ExitCode
        operator: In
        Values:
        - 42
```

Reply:
Update, in the current version of the API it is possible to express this as follows:

```
   - action: Terminate
     requirements:
     - onExitCodes:
        in:
        - 42
```